New/src/content/en/events/pwaroadshow/_index.yaml

```diff
@@ -0,0 +1,166 @@
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+landing_page:
+  custom_css_path: /web/events/events.css
+
+  header:
+    description: Progressive Web Apps leverage the best features of the web to create amazing offline experiences
+
+  rows:
+
+  - heading: Attend a Progressive Web App Roadshow in your area, <br> or start one yourself!
```

Comment:
I feel somewhat similar here as I did with the home page, this feels a bit like an orphaned heading, can we update the page header and merge the content here with find an event? Maybe retitle find an event?